### PR TITLE
Split common

### PR DESCRIPTION
**Pull request by joneff**
Split common into several files.

### CODE REVIEW

File: packages/default/scss/common/_typography.scss [hunk 1]
```diff
@@ -0,0 +1 @@
+@include exports( "common/typography" ) {}
```
Comment (gyoshev) [inline packages/default/scss/common/_typography.scss]: I'm still puzzled by placeholders. Why not add the file when it is needed?

File: packages/default/scss/common/utils/_flex.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
+@include exports( "common/utils/flex" ) {
```
Comment (gyoshev) [inline packages/default/scss/common/utils/_flex.scss]: Which component uses these utility classes?

Comment (joneff) [reply packages/default/scss/common/utils/_flex.scss]: For both questions the answer is the upcoming chat and card components

File: packages/default/scss/common/_typography.scss (hunk 1 above)
Comment (joneff) [reply packages/default/scss/common/_typography.scss]: I will need it for the card / chat components.

File: packages/default/scss/common/utils/_flex.scss (hunk 2 above)
Comment (gyoshev) [reply packages/default/scss/common/utils/_flex.scss]: Sounds good. In that case I suggest adding these classes along with the PR for the chat and card components, along with the example usage, and using a `feat:` commit to notify customers of the new theme features.

Comment (joneff) [reply packages/default/scss/common/utils/_flex.scss]: Done

**Pull request merged**